discovery-provider/compose/.env

```diff
@@ -12,13 +12,18 @@ audius_delegate_owner_wallet=0x1D9c77BcfBfa66D37390BF2335f0140979a6122B
 audius_delegate_private_key=0x3873ed01bfb13621f9301487cc61326580614a5b99f3c33cf39c6f9da3a19cad
 audius_discprov_identity_service_url=http://audius-identity-service_identity-service_1:7000
 audius_discprov_user_metadata_service_url=http://cn1_creator-node_1:4000
-audius_discprov_blacklist_block_indexing_interval=5
-audius_eth_contracts_registry=0x26b9c682a87879Ef035c92E4F978c303eC69a84B
-audius_solana_track_listen_count_address=4ZN1TTLp76gGCZ7tzXLfV4cahuHaP6oHFuiVWQS3knTp
+audius_eth_contracts_registry=0xABbfF712977dB51f9f212B85e8A4904c818C2b63
```

Comment:
did this legitimately change after pre-baked container?

Reply:
will revert oops